opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/Propagators.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Rename to `OpenTracingPropagators` to match `OpenTracingShim`?

Reply:
done